TodoList/sources/TodoList.Api/Controllers/NodesController.cs

```diff
@@ -114,13 +114,9 @@ private void ValidatePutNodeModel(Guid id, NodeModel node)
                 ModelState.AddModelError(nameof(node), "Node model is not correctly defined.");
                 return;
             }
-            if (node.Id == Guid.Empty)
-            {
-                ModelState.AddModelError(nameof(node.Id), "Node model must have id parameter specified here.");
-            }
-            if (id != node.Id)
+            if (node.Id != Guid.Empty)
```

Comment:
It's interesting you mostly use `default(Guid)`, but here you use `Guid.Empty`. Makes no difference, though ;)